restructuredtext_lint/sphinx.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import absolute_import
+
+import contextlib
+
+try:
+    import sphinx  # noqa
+
+    from sphinx.domains import c as c_domain
+    from sphinx.domains import cpp as cpp_domain
+    from sphinx.domains import javascript as javascript_domain
+    from sphinx.domains import python as python_domain
+    from sphinx.domains import std as std_domain
```

Comment:
Oooh, there seems to be `BUILTIN_DOMAINS` in Sphinx. Can we use that instead of hardcoding these imports?

https://github.com/sphinx-doc/sphinx/blob/1.3/sphinx/domains/__init__.py#L285-L292